### ISSUE TITLE
ClassNotFoundException for android.view.Button with TimePickerDialog on Android 9 + 8

### ISSUE THREAD

**Issue by Krillsson** [phrase/phrase-android#28]
We get weird exceptions when using Phrase SDK and `android.app.TimePickerDialog` widget.

Only reproduced it on Android 9 and 8 for now. 10 and above works as expected.

It seems to be caused by an older version of the ViewPump library. There is an identical issue in [this github project](https://github.com/B3nedikt/restring/issues/77).

Additionally, I wasn't able to test with 3.0.4 because something is broken with it on[ jitpack.io](https://jitpack.io/#phrase/phrase-android)

```kotlin
    android.view.InflateException: Binary XML file line #37: Binary XML file line #37: Error inflating class Button
    Caused by: android.view.InflateException: Binary XML file line #37: Error inflating class Button
    Caused by: java.lang.ClassNotFoundException: Didn't find class "android.view.Button" on path: DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/our.package-7ocexMTXPn9pTVBEZu4IXQ==/base.apk"],nativeLibraryDirectories=[/data/app/our.package-7ocexMTXPn9pTVBEZu4IXQ==/lib/x86_64, /system/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at android.view.LayoutInflater.createView(LayoutInflater.java:606)
        at android.view.LayoutInflater.onCreateView(LayoutInflater.java:703)
        at android.view.LayoutInflater.onCreateView(LayoutInflater.java:720)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:788)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:730)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:863)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
        at android.view.ViewStub.inflateViewNoAdd(ViewStub.java:269)
        at android.view.ViewStub.inflate(ViewStub.java:302)
        at com.android.internal.app.AlertController.resolvePanel(AlertController.java:506)
        at com.android.internal.app.AlertController.setupView(AlertController.java:530)
        at com.android.internal.app.AlertController.installContent(AlertController.java:264)
        at android.app.AlertDialog.onCreate(AlertDialog.java:436)
        at android.app.Dialog.dispatchOnCreate(Dialog.java:407)
        at android.app.Dialog.show(Dialog.java:302)
        at android.app.TimePickerDialog.show(TimePickerDialog.java:149)
        at androidx.fragment.app.DialogFragment.onStart(DialogFragment.java:688)
        at androidx.fragment.app.Fragment.performStart(Fragment.java:3008)
        at androidx.fragment.app.FragmentStateManager.start(FragmentStateManager.java:586)
        at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:300)
        at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2177)
        at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2088)
        at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:1990)
        at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:524)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6669)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
```

**Comment by theSoenke** [comment]
Hey @Krillsson, which version are of the SDK are you currently using? Are you already using 3.0.3? And could you also send me the error you're seeing from Jitpack? Installing from Jitpack works for me right now.

**Comment by theSoenke** [comment]
That's interesting where those build logs are coming from. Those look like quite old logs from another version. Right now the build has been successful https://jitpack.io/com/github/phrase/android-sdk/3.0.4/build.log

Could you give it one more try to install 3.0.4 now?

**Comment by Krillsson** [comment]
Ah. I checked the wrong address jitpack project (https://jitpack.io/#phrase/phrase-android).

Managed to get 3.0.4 now.

**Comment by Krillsson** [comment]
Still crashes on 3.0.4 though. Probably you are already aware of this.

**Comment by theSoenke** [comment]
@Krillsson We just released version 3.0.5 of the SDK, which includes a fix for this issue.
